### ISSUE TITLE
[Bug][A11Y] Checkbox item + icon : wrong accessible name

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#392]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Android and iOS

### OS Version

.

### App version

0.6.0

### Describe the issue

For checkbox item component, if we add the icon setting, the button role is added to the checkbox one

**Android - extra role "button is heard:** "not checked, checkbox, label, **button**, double-tap to activate"

**iOS - image is read:** "checkbox, label, not checked, **image**"

### Expected Behavior

- Remove the button role on Android
- Set the image as decorative on iOS

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (457)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 430
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (alpha) (457)
Version: 0.6.0
Build type: Alpha
Build details (GitHub): 430
cc @B3nz01d (product owner)
cc @Tayebsed93 @nouha06 @AhmedAmineZr (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by B3nz01d** [comment]
@Tayebsed93  & @nouha06 for the moment the iOS version is not yet available

For Android, I Need to perform 2 swipes before I get to the checkbox, I think the accessibility item must be put at 2 levels and should only be setup to the one where the entire checkbox information is read

**Comment by nouha06** [comment]
> [@Tayebsed93](https://github.com/Tayebsed93) & [@nouha06](https://github.com/nouha06) for the moment the iOS version is not yet available
> 
There is currently a problem with iOS build deployments

**Comment by nouha06** [comment]
I fixed this issue, we currently have a deployment problem for iOS on TestFlight, so I’ll deliver this fix later.
Added blocking labels as deployment to TestFlight is not possible at the moment.

**Comment by pya35** [comment]
@nouha06 on Android 06.0(470), as Benoit said, you should remove the first accessible item heard "not ticked, tickbox"

<img width="395" height="830" alt="" src="https://github.com/user-attachments/assets/810e9da8-be96-4750-9567-2b86a2c3ef3e" />

**Comment by nouha06** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 430
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 430
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)
